oaknational/Oak-Web-Application
# refactor: adds next seo through layout component

## Description

- Adds next seo through layout component props
- seoProps is required prop
- default props can be imported - `import { DEFAULT_SEO_PROPS } from "../browser-lib/seo/Seo";`

`  <Layout seoProps={DEFAULT_SEO_PROPS}>`

- Or create custom props
```
     <Layout
        seoProps={{
          title: `${lesson.title} lesson | Oak National Academy`,
          description:
            "This lesson revises the Forces subject knowledge of the GCSE Physics Science only, and gives an opportunity to work through some independent tasks and exam questions.",
        }}
      >
```


## Issue(s)

- Prompts a reminder to consider seo when new page is created
- No longer have to import seo into individual page

## How to test

1. Go to {[cloud link](https://samara-m0gd474k2-oak-national-academy.vercel.app/lessons/physics-only-review-chj3cd)}
- Inspect with google, you can see custom SEO for lesson page in the head

2. Go to [cloud link](https://samara-m0gd474k2-oak-national-academy.vercel.app/)
- Inspect with google, you can see default SEO


## Checklist
